clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -64,6 +64,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(ConsumerCoordinator.class);
 
+    private static final long CLOSE_TIMEOUT_MS = 5000;
```

Comment:
Should we hard code this or add a new method close(long timeout, TimeUnit timeUnit) like the producer?